includes/definitions/proxmox.yaml

```diff
@@ -0,0 +1,25 @@
+os: proxmox
+type: server
+group: unix
+text: proxmox pve
+icon: proxmox
+ifXmcbc: 1
+ifname: 1
+over:
+    - { graph: device_processor, text: 'Processor Usage' }
+    - { graph: device_ucd_memory, text: 'Memory Usage' }
+    - { graph: device_storage, text: 'Storage Usage' }
+poller_modules:
+    bgp-peers: 0
+    ospf: 0
+    stp: 0
+discovery_modules:
+    applications: 1
+    bgp-peers: 0
+    stp: 0
+    vmware-vminfo: 0
+    libvirt-vminfo: 0
+processor_stacked: 1
+discovery:
+    - sysDescr_regex:
+        - '/PVE/'
```

Comment:
This doesn't need to be a regex.  You are just searching for PVE anywhere in the sysDescr.
```
 - sysDescr: PVE
```